From c935649d4db8e1433dfbbd14692a382877107d1a Mon Sep 17 00:00:00 2001
From: Joerg Behrmann <behrmann@physik.fu-berlin.de>
Date: Wed, 22 May 2019 10:55:48 +0200
Subject: [PATCH] mkosi: provide default xbootldr_partno in argument parsing

Fixes #333
---
 mkosi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mkosi b/mkosi
index 7551d02..22ab7f7 100755
--- a/mkosi
+++ b/mkosi
@@ -91,6 +91,7 @@ class CommandLineArguments(argparse.Namespace):
 
     swap_partno: Optional[int] = None
     esp_partno: Optional[int] = None
+    xbootldr_partno: Optional[int] = None
 
 
 class SourceFileTransfer(enum.Enum):
-- 
2.22.0

